Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.h

```diff
@@ -105,8 +105,8 @@ class WebAuthenticatorCoordinatorProxy : public IPC::MessageReceiver {
 
 #if HAVE(WEB_AUTHN_AS_MODERN)
     RetainPtr<ASAuthorizationController> constructASController(WebAuthenticationRequestData&&);
-    RetainPtr<NSArray> requestsForRegisteration(const WebCore::PublicKeyCredentialCreationOptions&, const Vector<uint8_t>& hash);
-    RetainPtr<NSArray> requestsForAssertion(const WebCore::PublicKeyCredentialRequestOptions&, const Vector<uint8_t>& hash, std::optional<WebCore::SecurityOriginData>& parentOrigin);
+    RetainPtr<NSArray> requestsForRegisteration(const WebCore::PublicKeyCredentialCreationOptions&, const Vector<uint8_t>& hash, const String&);
```

Comment:
Typo: requestsForRegisteration
Also, please don't omit the new parameter's name since it is not obvious what it is from its type.

Reply:
Done (omitting the name was not intentional)